app/src/test/java/edu/ucandroid/weathernotice/CountryUnitTest.kt

```diff
@@ -28,12 +28,6 @@ class ExampleUnitTest {
 
     }
 
-    @Test
-    fun addition_isCorrect() {
-        assertEquals(4, 2 + 2)
-    }
-
-
```

Comment:
Seems like a leftover test method to see how they work, as it doesn't touch any code.

Reply:
Good catch.